modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+[id="cnf-scheduling-workload-onto-worker-with-real-time-capabilities_{context}"]
+= Scheduling a workload onto a worker with real-time capabilities
+
+You can schedule realtime workloads onto a worker node where a performance profile that configures real-time capabilities is applied.
```

Comment:
I wouldn't say "realtime" workload - linux doesn't support true realtime workloads. I think it is better to call these "low latency" workloads. Comment applies everywhere (assuming other folks agree with me).